### PR TITLE
WIP: New Colors 4: Desktop inbox

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 81725f6a6fbe...3fb0a5f0c874 on linux. <details><summary>:mag_right: 6 errored, 12 changed</summary>

 * Error in 81725f6a6fbe: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 3fb0a5f0c874: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 3fb0a5f0c874: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Error in 3fb0a5f0c874: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Changed: **Chat_Side_Panel-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Side_Panel-Normal-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Side_Panel-Normal-3fb0a5f0c874.png)  
   Chat_Side_Panel-Normal comparison
 * Changed: **Chat_Side_Panel-Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Side_Panel-Muted-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Side_Panel-Muted-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Side_Panel-Muted.png)
 * Changed: **Chat_Conversations_List-You_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-You_Rekey_Selected-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-You_Rekey_Selected-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-You_Rekey_Selected.png)
 * Changed: **Chat_Conversations_List-You_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-You_Rekey-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-You_Rekey-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-You_Rekey.png)
 * Changed: **Chat_Conversations_List-Selected_Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Selected_Normal-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Selected_Normal-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Selected_Normal.png)
 * Changed: **Chat_Conversations_List-Selected_Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Selected_Muted-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Selected_Muted-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Selected_Muted.png)
 * Changed: **Chat_Conversations_List-Part_Rekey_Selected** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Part_Rekey_Selected-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Part_Rekey_Selected-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Part_Rekey_Selected.png)
 * Changed: **Chat_Conversations_List-Part_Rekey** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Part_Rekey-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Part_Rekey-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Part_Rekey.png)
 * Changed: **Chat_Conversations_List-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Normal-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Normal-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Normal.png)
 * Changed: **Chat_Conversations_List-Long_Top** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Long_Top-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Long_Top-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Long_Top.png)
 * Changed: **Chat_Conversations_List-Long_Bottom** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Long_Bottom-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Long_Bottom-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Long_Bottom.png)
 * Changed: **Chat_Conversations_List-Empty** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Empty-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Empty-3fb0a5f0c874.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497995821223-81725f6a6fbe-3fb0a5f0c874-linux/Chat_Conversations_List-Empty.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 81725f6a6fbe...8d6a3fa68dcb on linux. <details><summary>:mag_right: 6 errored, 11 new, 10 removed, 4 changed</summary>

 * Error in 81725f6a6fbe: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 8d6a3fa68dcb: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 8d6a3fa68dcb: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Error in 8d6a3fa68dcb: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Added: **Chat_No_Conversation-normal**  
   Chat_No_Conversation-normal rendered
 * Added: **Chat_Inbox-You_Rekey_Selected** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-You_Rekey_Selected-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-You_Rekey** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-You_Rekey-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-Selected_Normal** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-Selected_Normal-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-Selected_Muted** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-Selected_Muted-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-Part_Rekey_Selected** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-Part_Rekey_Selected-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-Part_Rekey** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-Part_Rekey-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-Normal** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-Normal-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-Long_Top** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-Long_Top-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-Long_Bottom** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-Long_Bottom-8d6a3fa68dcb.png)
 * Added: **Chat_Inbox-Empty** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Inbox-Empty-8d6a3fa68dcb.png)
 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Tracker-New_user_Scroll_1-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Tracker-New_user_Scroll_1-8d6a3fa68dcb.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Tracker-New_user_Scroll_1.png)
 * Changed: **Icon-Icon_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Icon-Icon_Image-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Icon-Icon_Image-8d6a3fa68dcb.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Icon-Icon_Image.png)
 * Changed: **Chat_Side_Panel-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Side_Panel-Normal-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Side_Panel-Normal-8d6a3fa68dcb.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Side_Panel-Normal.png)
 * Changed: **Chat_Side_Panel-Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Side_Panel-Muted-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Side_Panel-Muted-8d6a3fa68dcb.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Side_Panel-Muted.png)
 * Removed: **Chat_Conversations_List-You_Rekey_Selected** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-You_Rekey_Selected-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-You_Rekey** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-You_Rekey-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Selected_Normal** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-Selected_Normal-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Selected_Muted** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-Selected_Muted-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Part_Rekey_Selected** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-Part_Rekey_Selected-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Part_Rekey** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-Part_Rekey-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Normal** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-Normal-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Long_Top** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-Long_Top-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Long_Bottom** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-Long_Bottom-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Empty** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997043897-81725f6a6fbe-8d6a3fa68dcb-linux/Chat_Conversations_List-Empty-81725f6a6fbe.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
:no_entry: Error rendering the commits 81725f6a6fbe...652876123acd on linux. <details><summary>:mag_right: 6 errored, 11 new, 10 removed, 4 changed</summary>

 * Error in 81725f6a6fbe: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 652876123acd: **Signup_Passphrase-Start-ERROR**  
   Signup_Passphrase-Start-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 652876123acd: **Signup_Passphrase-Error-ERROR**  
   Signup_Passphrase-Error-ERROR rendered
 * Error in 81725f6a6fbe: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Error in 652876123acd: **Signup_Error-Start-ERROR**  
   Signup_Error-Start-ERROR rendered
 * Added: **Chat_No_Conversation-normal**  
   Chat_No_Conversation-normal rendered
 * Added: **Chat_Inbox-You_Rekey_Selected** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-You_Rekey_Selected-652876123acd.png)
 * Added: **Chat_Inbox-You_Rekey** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-You_Rekey-652876123acd.png)
 * Added: **Chat_Inbox-Selected_Normal** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-Selected_Normal-652876123acd.png)
 * Added: **Chat_Inbox-Selected_Muted** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-Selected_Muted-652876123acd.png)
 * Added: **Chat_Inbox-Part_Rekey_Selected** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-Part_Rekey_Selected-652876123acd.png)
 * Added: **Chat_Inbox-Part_Rekey** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-Part_Rekey-652876123acd.png)
 * Added: **Chat_Inbox-Normal** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-Normal-652876123acd.png)
 * Added: **Chat_Inbox-Long_Top** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-Long_Top-652876123acd.png)
 * Added: **Chat_Inbox-Long_Bottom** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-Long_Bottom-652876123acd.png)
 * Added: **Chat_Inbox-Empty** [(view)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Inbox-Empty-652876123acd.png)
 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Tracker-New_user_Scroll_1-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Tracker-New_user_Scroll_1-652876123acd.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Tracker-New_user_Scroll_1.png)
 * Changed: **Icon-Icon_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Icon-Icon_Image-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Icon-Icon_Image-652876123acd.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Icon-Icon_Image.png)
 * Changed: **Chat_Side_Panel-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Side_Panel-Normal-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Side_Panel-Normal-652876123acd.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Side_Panel-Normal.png)
 * Changed: **Chat_Side_Panel-Muted** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Side_Panel-Muted-81725f6a6fbe.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Side_Panel-Muted-652876123acd.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Side_Panel-Muted.png)
 * Removed: **Chat_Conversations_List-You_Rekey_Selected** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-You_Rekey_Selected-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-You_Rekey** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-You_Rekey-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Selected_Normal** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-Selected_Normal-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Selected_Muted** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-Selected_Muted-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Part_Rekey_Selected** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-Part_Rekey_Selected-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Part_Rekey** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-Part_Rekey-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Normal** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-Normal-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Long_Top** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-Long_Top-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Long_Bottom** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-Long_Bottom-81725f6a6fbe.png)
 * Removed: **Chat_Conversations_List-Empty** [(view original)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1497997796654-81725f6a6fbe-652876123acd-linux/Chat_Conversations_List-Empty-81725f6a6fbe.png)
</summary>

File: shared/chat/inbox/index.desktop.js [hunk 1]
```diff
@@ -375,9 +374,11 @@ const conversationRowStyle = {
 
 const containerStyle = {
   ...globalStyles.flexBoxColumn,
-  backgroundColor: globalColors.darkBlue4,
+  backgroundColor: globalColors.white,
+  borderRight: `1px solid ${globalColors.black_05}`,
   flex: 1,
-  maxWidth: 240,
+  maxWidth: 241,
```
Comment (chromakode) [inline shared/chat/inbox/index.desktop.js]: Does setting `width` not suffice?

**Comment by chrisnojima** [comment]
I don't think so. With flexbox it won't constrain it so it'll grow. We
needed this kind of thing in several places. Would love a different
solution but this seemed simple

**Pull request closed without merging**